aireview/src/aireview/domain/entities/review_comment.py

```diff
@@ -4,6 +4,7 @@
 
 @dataclass
 class ReviewComment:
+    sha: str
```

Comment:
**style**: Adding sha field to ReviewComment class breaks backward compatibility. Consider making it optional with a default value or create a new version of the class.